README.md

```diff
@@ -23,3 +24,8 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+I had chat gpt help with solving Counter example [0,0] which had a lot of complicated naming and defining things which added some parts to the code and since this was the first jsverfiy test i had to run my self, i got help making that aswell.
```

Comment:
Not sure what you mean -- `[0,0]` isn't one of your tests? And you're not using jsverify.

Reply:
Oh ya sorry i changed to just general checks because me an Ali couldnt get that version working, Ill change that in my readme.  Im pretty sure i had the check for [0,0] but there was a lot of things going wrong so it might have been a different problem causing it.

Reply:
`[0,0]` is still not one of your tests, but if that fails it should be...

Reply:
when testing for [0,0] i would be expecting  to get [0,0] back right or would i just want [0]

Reply:
That depends on how you implemented it. It might also not be a valid input.

Reply:
I have updated it to what i believe is the correct implimentation